deployment/ecs-params.template.yml

```diff
@@ -0,0 +1,14 @@
+version: 1
```

Comment:
ditto

Reply:
Again, you need to know and understand the deployment process to ECS.

Reply:
the average joe just needs to know it's used in the ecs deploy shell script